modules/gateway/rpc.go

```diff
@@ -69,6 +69,19 @@ func (g *Gateway) RegisterRPC(name string, fn modules.RPCFunc) {
 	g.handlers[handlerName(name)] = fn
 }
 
+// UnregisterRPC unregisters an RPC and removes the corresponding RPCFunc from
+// g.handlers. Future calls to the RPC by peers will fail.
+func (g *Gateway) UnregisterRPC(name string) {
+	id := g.mu.Lock()
+	defer g.mu.Unlock(id)
+	if build.DEBUG {
+		if _, ok := g.handlers[handlerName(name)]; !ok {
+			panic("RPC is not registered: " + name)
+		}
+	}
```

Comment:
`build.Critical` -> panics `if DEBUG` otherwise prints a strong message to the terminal.